lib/workload/stateless/stacks/postgres-manager/deploy/stack.ts

```diff
@@ -9,13 +9,28 @@ import * as rds from 'aws-cdk-lib/aws-rds';
 import * as ssm from 'aws-cdk-lib/aws-ssm';
 import * as sm from 'aws-cdk-lib/aws-secretsmanager';
 import { MicroserviceConfig, DbAuthType } from '../function/type';
-import package_json from '../package.json';
```

Comment:
Oh, I missed this part with previous PR. Ah Haha 😁 All good. Now I got your point. It's better now with leveraging provider framework.